usaspending_api/decorators.py

```diff
@@ -0,0 +1,14 @@
+import logging
```

Comment:
Please store this in `usaspending-api/usaspending_api/common/api_versioning.py`

And change the header value to 
```WARNING! You are using a deprecated version of the API which maybe unstable and will be removed in the future.```